### PR TITLE
Highlight tabs

### PR DESCRIPTION
**Pull request by Brijendrayaduvanshi**
Highlight tabs

### CODE REVIEW

File: blocks/highlight-tabs/highlight-tabs.css [hunk 1]
```diff
@@ -186,9 +185,9 @@
   }
 }
 .highlightItem-content {
-  margin-left: 20px;
-  margin-top: 80px;
-  width: 85%;
+  position: relative;
+  margin-top: 24px;
+  width: calc(100%-20px);
```
Comment (amannar) [inline blocks/highlight-tabs/highlight-tabs.css]: is this getting applied? there should be a space around '-'

File: blocks/highlight-tabs/highlight-tabs.css [hunk 2]
```diff
@@ -35,20 +35,21 @@
   .highlightItem-img {
     object-fit: cover;
     width: 100%;
```
Comment (amannar) [inline blocks/highlight-tabs/highlight-tabs.css]: where is max-width: 100% on img tag?

Comment (Brijendrayaduvanshi) [reply blocks/highlight-tabs/highlight-tabs.css]: line no.37 is having width:100%

File: blocks/highlight-tabs/highlight-tabs.css (hunk 1 above)
Comment (Brijendrayaduvanshi) [reply blocks/highlight-tabs/highlight-tabs.css]: done

File: blocks/highlight-tabs/highlight-tabs.css (hunk 2 above)
Comment (Brijendrayaduvanshi) [reply blocks/highlight-tabs/highlight-tabs.css]: done

**Review by amannar — APPROVED** [review]

**Pull request merged**